apps/api/ecosystem.config.js

```diff
@@ -1,7 +1,7 @@
 module.exports = {
   apps: [
     {
-      name: 'osmo-notify', // Name of your application
+      name: 'osmoX', // Name of your application
```

Comment:
Update to OsmoX

Reply:
Addressed in https://github.com/OsmosysSoftware/osmo-x/pull/152/commits/ffca8627cb4ae9663e16c6d1288440193d1ada1b